src/HealthGPS.Console/model_parser.cpp

```diff
@@ -146,64 +146,41 @@ load_staticlinear_risk_model_definition(const poco::json &opt, const host::Confi
     const auto correlations_table = load_datatable_from_csv(correlations_file_info);
     Eigen::MatrixXd correlations{correlations_table.num_rows(), correlations_table.num_columns()};
 
-    // Weight quantiles.
-    const auto quantiles_female = load_datatable_from_csv(
-        host::get_file_info(opt["WeightQuantiles"]["female"], config.root_path));
-    const auto quantiles_male = load_datatable_from_csv(
-        host::get_file_info(opt["WeightQuantiles"]["male"], config.root_path));
-    std::map<hgps::core::Gender, std::vector<double>> weight_quantiles = {
-        {hgps::core::Gender::female, {}}, {hgps::core::Gender::male, {}}};
-    weight_quantiles[hgps::core::Gender::female].reserve(quantiles_female.num_rows());
-    weight_quantiles[hgps::core::Gender::male].reserve(quantiles_male.num_rows());
-    for (size_t j = 0; j < quantiles_female.num_rows(); j++) {
-        weight_quantiles[hgps::core::Gender::female].push_back(
-            std::any_cast<double>(quantiles_female.column(0).value(j)));
-    }
-    for (size_t j = 0; j < quantiles_male.num_rows(); j++) {
-        weight_quantiles[hgps::core::Gender::male].push_back(
-            std::any_cast<double>(quantiles_male.column(0).value(j)));
-    }
+    // Risk factor names, models, parameters and correlation matrix.
+    std::vector<hgps::core::Identifier> names{};
+    std::vector<hgps::LinearModelParams> models{};
+    std::vector<double> lambda{};
+    std::vector<double> stddev{};
```

Comment:
These will already be default-initialised, so you don't need the curlies:

```suggestion
    std::vector<hgps::core::Identifier> names;
    std::vector<hgps::LinearModelParams> models;
    std::vector<double> lambda;
    std::vector<double> stddev;
```

You only need those for basic types like ints which will otherwise be uninitialised.

Reply:
Ah, okay. I will fix these in the next PR, if that's okay with you.